### ISSUE TITLE
Guild init with --python flag

### ISSUE THREAD

**Issue by samedii** [guildai/guildai#114]
This is not working anymore in rc4, it was working in rc2

    guild init -p 3.6

```
You are about to initialize a Guild environment:
  Location: ~/Documents/rameye/venv
  Name: rameye
  Python interpreter: python3.6
  Guild: 0.7.0.rc4
  Python requirements:
    ./requirements.txt
  Resource cache: shared
Continue? (Y/n) 
Initializing Guild environment in /home/richard/Documents/rameye/venv
Resource cache /home/richard/Documents/rameye/venv/.guild/cache/resources exists, skipping
Creating virtual environment
usage: venv [-h] [--system-site-packages] [--symlinks | --copies] [--clear]
            [--upgrade] [--without-pip] [--prompt PROMPT]
            ENV_DIR [ENV_DIR ...]
venv: error: unrecognized arguments: --python python3.6
guild: Command '['/usr/bin/python3', '-m', 'venv', '/home/richard/Documents/rameye/venv', '--prompt', '(rameye) ', '--python', 'python3.6']' returned non-zero exit status 2.

```

**Comment by samedii** [comment]
Reverted to rc3, also failing there

**Comment by gar1t** [comment]
Ah, thank you! Will get this fixed asap.

**Comment by gar1t** [comment]
The fix is now generally available in 0.7.0. If this is still an issue, please re-open and we'll sort it out. Thanks again for the report.
